### PR TITLE
Misc edits to training

### PR DESCRIPTION
**Pull request by lhirata**

### CODE REVIEW

File: training/back-up-a-cluster.md [hunk 1]
```diff
@@ -29,19 +29,22 @@ $ cockroach gen example-data | cockroach sql --certs-dir=certs
 ### Step 2. Enable license
 
 1. Open the SQL shell:
-	
+
 	{% include copy-clipboard.html %}
```
Comment (jseldess) [inline training/back-up-a-cluster.md]: Indented too far.

File: training/back-up-a-cluster.md [hunk 2]
```diff
@@ -29,19 +29,22 @@ $ cockroach gen example-data | cockroach sql --certs-dir=certs
...
+
+    {% include copy-clipboard.html %}
     ~~~ sql
-    > SET CLUSTER SETTING cluster.organization = 'Cockroach Labs Training'
-    
-    > SET CLUSTER SETTING enterprise.license = '{{site.training.ccl_license}}'
+    > SET CLUSTER SETTING cluster.organization = 'Cockroach Labs Training';
+    ~~~
+
+		{% include copy-clipboard.html %}
```
Comment (jseldess) [inline training/back-up-a-cluster.md]: This and next line are indented too far.

File: training/back-up-a-cluster.md [hunk 3]
```diff
@@ -29,19 +29,22 @@ $ cockroach gen example-data | cockroach sql --certs-dir=certs
...
+		~~~ sql
```
Comment (jseldess) [inline training/back-up-a-cluster.md]: Indented too far.

File: training/decommission-nodes.md [hunk 4]
```diff
@@ -48,7 +49,7 @@ Because our cluster has a replication factor of 3, we must add a fourth node to
     ~~~
 
 3. Make sure the fourth node joined the cluster:
-	
+
 	{% include copy-clipboard.html %}
```
Comment (jseldess) [inline training/decommission-nodes.md]: Indented too far.

Comment (lhirata) [reply training/decommission-nodes.md]: done

File: training/back-up-a-cluster.md (hunk 3 above)
Comment (lhirata) [reply training/back-up-a-cluster.md]: done

File: training/back-up-a-cluster.md (hunk 2 above)
Comment (lhirata) [reply training/back-up-a-cluster.md]: done

File: training/back-up-a-cluster.md (hunk 1 above)
Comment (lhirata) [reply training/back-up-a-cluster.md]: done

**Comment by sploiselle** [comment]
This is great. Thank you so much for taking the time to improve this, @lhirata!

**Pull request merged**